demisto_sdk/commands/update_release_notes/update_rn_manager.py

```diff
@@ -101,17 +107,13 @@ def handle_api_module_change(self, modified_files: set, added_files: set):
                 added_files: A set of new added files
                 modified_files: A set of modified files
         """
-        # The user gave a path to the api module which was changed
-        if self.given_pack and API_MODULES_PACK in self.given_pack:
-            update_api_modules_dependents_rn(self.pre_release, self.update_type, added_files,
-                                             modified_files, self.id_set_path, self.text)
-
-        # The user didn't give a path to the api module but they have changed.
-        elif self.changed_packs_from_git and API_MODULES_PACK in self.changed_packs_from_git:
-            if click.confirm('Some API modules seem to have changed, '
```

Comment:
Didnt we say we will keep it as a message?
Just inform the user about it but not ask

Reply:
Done